### PR TITLE
Better filter: `issue type` in `JiraChanges`

### PR DESCRIPTION
**Pull request by ghost**
- `issue_type` include only options relavant for jira-changes
- fixed 'planed' typo
- better look for search form
- help text for `ci` filer field saying it filters either `name` and `uid`

### CODE REVIEW

File: src/ralph/cmdb/forms.py [hunk 1]
```diff
@@ -410,7 +411,7 @@ class ReportFiltersDateRange(forms.Form):
     start_planned_start = forms.DateField(
         required=False,
         widget=DateWidget(dict(date_attrs, placeholder='From')),
-        label="Planed start",
+        label="Planned start",
```
Comment (ar4s) [inline src/ralph/cmdb/forms.py]: use _

File: src/ralph/cmdb/forms.py [hunk 2]
```diff
@@ -311,6 +311,7 @@ class ReportFilters(forms.Form):
 
     ci = forms.CharField(
         required=False,
+        help_text=_('searched by "name" or "uid"'),
```
Comment (mkurek) [inline src/ralph/cmdb/forms.py]: just `"name" or "uid"`

File: src/ralph/cmdb/forms.py [hunk 3]
```diff
@@ -311,6 +311,7 @@ class ReportFilters(forms.Form):
...
         widget=forms.TextInput(
             attrs=dict(input_attrs, placeholder='ralph'),
```
Comment (mkurek) [inline src/ralph/cmdb/forms.py]: remove this placeholder (and asignee placeholder too)

File: src/ralph/cmdb/forms.py [hunk 4]
```diff
@@ -420,7 +421,7 @@ class ReportFiltersDateRange(forms.Form):
     start_planned_end = forms.DateField(
         required=False,
         widget=DateWidget(dict(date_attrs, placeholder='From')),
-        label="Planed end",
+        label="Planned end",
```
Comment (mkurek) [inline src/ralph/cmdb/forms.py]: fix placeholders consistency (from-to, start-end etc)

**Comment by mkurek** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**